AppDevAnnouncements/Classes/AnnouncementNetworking.swift

```diff
@@ -52,25 +51,24 @@ public class AnnouncementNetworking {
 }
 
 internal extension UIImageView {
-    func loadFromURL(photoUrl: String, completion: @escaping ((Bool) -> Void) = {_ in}) {
-        guard let url = URL(string: photoUrl) else {completion(false); return}
+
+    func loadFrom(photoUrl: String, completion: ((Bool) -> Void)?) {
```

Comment:
I feel like we could probably replace `photoUrl` with just `url`, but other than that looks good.

Reply:
I was thinking that we should specify what kind of URL it is, but then realized this is inside an imageview extension lol, I'll change that right now